packages/example/src/design-system/components/Arrows.tsx

```diff
@@ -0,0 +1,50 @@
+import styled from '@emotion/native';
+import { Image } from 'react-native';
+import arrowSource from '../assets/arrow-left.png';
+
+const LeftArrowImage = styled(Image)({
+  height: 70,
+  width: 50,
+  resizeMode: 'stretch',
+  transform: [{ rotate: '180deg' }],
+  tintColor: 'white',
+});
+
+const RightArrowImage = styled(Image)({
+  height: 70,
+  width: 50,
+  resizeMode: 'stretch',
+  tintColor: 'white',
+});
+
+export const LeftArrow = () => {
```

Comment:
You can memoise these easily, they have no props so they're very easy to memoize 😁 

```suggestion
export const LeftArrow = React.memo(() => {
```

Reply:
Fixed